unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -222,99 +225,99 @@ public bool ProcessMessage(MessagingBus.QueuedSceneMessage_Scene msgObject, out
                 switch (method)
                 {
                     case MessagingTypes.ENTITY_CREATE:
-                    {
-                        if (msgObject.payload is Protocol.CreateEntity payload)
-                            scene.CreateEntity(payload.entityId);
+                        {
+                            if (msgObject.payload is Protocol.CreateEntity payload)
+                                scene.CreateEntity(payload.entityId);
 
-                        break;
-                    }
+                            break;
+                        }
                     case MessagingTypes.ENTITY_REPARENT:
-                    {
-                        if (msgObject.payload is Protocol.SetEntityParent payload)
-                            scene.SetEntityParent(payload.entityId, payload.parentId);
+                        {
+                            if (msgObject.payload is Protocol.SetEntityParent payload)
+                                scene.SetEntityParent(payload.entityId, payload.parentId);
```

Comment:
Any reason for this extra indentation?

Reply:
the auto formatter file we have in the repo should be responsible for that, probably we have 2 different ones, one for VSCode and another for Rider? or where do you guys store your formatting rules?

Reply:
We dont have a common one, we should though. @BrianAmadori